### PR TITLE
Build fixes; got working on WIndows

### PR DESCRIPTION
**Pull request by LarryOsterman**

### CODE REVIEW

File: src/curl_easy.cpp [hunk 1]
```diff
@@ -84,6 +88,27 @@ std::unique_ptr<curl_easy> curl_easy::create(const std::string& url)
...
+    char fileNameBuffer[MAX_PATH];
+    if (GetModuleFileNameA(moduleHandle, fileNameBuffer, _countof(fileNameBuffer)) == 0)
+    {
+        throw std::exception("Unable to retrieve module name.");
+    }
+    if (!PathRemoveFileSpecA(fileNameBuffer))
+    {
+        throw std::exception("Unable to remove filename from buffer.");
+    }
+    if (FAILED(StringCchCatA(fileNameBuffer, _countof(fileNameBuffer), "\\curl-ca-bundle.crt")))
```
Comment (soccerGB) [inline src/curl_easy.cpp]: I am wondering why a fix file name is used here.
Is this for testing purpose?

Comment (anitagov) [reply src/curl_easy.cpp]: Yes, this is the file from copied from libcurl.

**Comment by anitagov** [comment]
Adding @CodeMonkeyLeet ...

File: src/curl_easy.cpp (hunk 1 above)
Comment (LarryOsterman) [reply src/curl_easy.cpp]: LibCURL requires this filename.

**Pull request merged**